server/food_bank_app/inventory/selectors.py

```diff
@@ -0,0 +1,10 @@
+from .models import Inventory
+
+def select_inventories_by_date(date):
+    try:
+        inventories = Inventory.objects.filter(year=date.year, month=date.month)
+    except Inventory.DoesNotExist:
+        inventory = None
```

Comment:
```suggestion
        inventories = []
```
Altuough I think objects.filter already does that

Reply:
Although*